src/js/ripple/log.js

```diff
@@ -149,7 +149,11 @@ var NullLogEngine = {
 if (typeof window !== 'undefined' && typeof console !== 'undefined') {
   Log.engine = InteractiveLogEngine;
 } else if (typeof console !== 'undefined' && console.log) {
-  Log.engine = BasicLogEngine;
+  if (process.env.NODE_ENV === 'test') {
+    Log.engine = NullLogEngine;
```

Comment:
The only real "problem" was vague error messages on Remote._handleMessage, which I did actually notice, and referenced earlier in this pull request (https://github.com/ripple/ripple-lib/issues/307). Ostensibly this pull request (328) regards unnecessary logging during a test for error condition, which caught someone else's attention because the log was poor, non-descriptive